src/client/utils/url-resolver.js

```diff
@@ -53,8 +54,10 @@ export default {
             // NOTE: It looks like a Chrome bug: in a nested iframe without src (when an iframe is placed into another
             // iframe) you cannot set a relative link href while the iframe loading is not completed. So, we'll do it with
             // the parent's urlResolver Safari demonstrates similar behavior, but urlResolver.href has a relative URL value.
-            const needUseParentResolver = url && isIframeWithoutSrc && window.parent && window.parent.document &&
-                                          (!href || href.indexOf('/') === 0);
+            const isIframeWithUnresolvedUrl = url && window.parent && window.parent !== window &&
```

Comment:
It looks like an intersection or bad variable naming.
You 3 times check that current window is iframe. 
Try to refactor.